### DISCUSSION TITLE
Help for Lib BlynkGSM_Manager

### DISCUSSION THREAD

**Discussion by Guiimartinho** [khoih-prog/ESP_WiFiManager#76] [Q&A]
Hello how are you?
I downloaded everything from the BlynkGSM_Manager library, and I would to ask you if possible!

I'm developing an academic work, and I need to use GSM together with WIFI, but without the blynk interface, but an interface that I developed.

I would like to ask you any questions

**Comment by khoih-prog** [comment]
Without Blynk must be much **easier** to use and code.

Have a look at [**BlynkSimpleEsp32_GSM_WFM.h**](https://github.com/khoih-prog/BlynkGSM_Manager/blob/master/src/BlynkSimpleEsp32_GSM_WFM.h), and just **remove any code related to Blynk.**

You must have basic programming knowledge and skills to do the coding.

I'm sorry I don't have time to help here. Please ask in **Arduino** or **ESP32** Forum for help as I don't know if anybody here can help write the code for you.

Good Luck,
